src/uproot/models/TTree.py

```diff
@@ -905,3 +905,69 @@ def read_members(self, chunk, cursor, context, file):
 
 uproot.classes["TTree"] = Model_TTree
 uproot.classes["ROOT::TIOFeatures"] = Model_ROOT_3a3a_TIOFeatures
+
+
+fEntriesStruct = struct.Struct(">q")
+
+
+class Model_TTree_NumEntries(uproot.model.Model):
+    """
+    A helper :doc:`uproot.model.Model` for :doc:`uproot.num_entries`.
+    """
+
+    def read_members(self, chunk, cursor, context, file):
+        if self.is_memberwise:
+            raise NotImplementedError(
+                """memberwise serialization of {}
+in file {}""".format(
+                    type(self).__name__, self.file.file_path
+                )
+            )
+        cursor.skip_over(chunk, context)
+        cursor.skip_over(chunk, context)
+        cursor.skip_over(chunk, context)
+        cursor.skip_over(chunk, context)
```

Comment:
I'm just giving this some documentation, to explain why we skip exactly 4 times.

```suggestion
        cursor.skip_over(chunk, context)  # TNamed
        cursor.skip_over(chunk, context)  # TAttLine
        cursor.skip_over(chunk, context)  # TAttFill
        cursor.skip_over(chunk, context)  # TAttMarker
```